### DISCUSSION TITLE
Moved Install...Login to Admin works, but links don't redirect

### DISCUSSION THREAD

**Discussion by adamesq** [YOURLS/YOURLS#3583] [Q&A] [answered]
I copied over the install and changed the config file. 

Everything looks good in the admin panel, but the links aren't getting redirected...they just try to go to the shortened URL itself.

Did I make a noobie error?

**Comment by dgw** [comment]
Incomplete copy? You need the `.htaccess` file, or to copy the configuration for Nginx/Caddy/whatever other server you use if not Apache.

**Reply by adamesq** [reply]
Thank you for the quick reply, @dgw. Yes, I copied over the .htaccess file as well as the config (same server).

**Comment by adamesq** [comment, marked as answer]
SOLVED! Yup....I made a noobie-type error! Nothing to see here...thank you for offering to help to anyone who drops in. Misconfigured the .htaccess redirect.
